packages/lisk-transactions/src/4_register_multisignature_account.js

```diff
@@ -12,16 +12,52 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import { MULTISIGNATURE_FEE } from './constants';
+import {
+	MULTISIGNATURE_FEE,
+	MULTISIGNATURE_MAX_LIFETIME,
+	MULTISIGNATURE_MIN_LIFETIME,
+	MULTISIGNATURE_MAX_KEYSGROUP,
+	MULTISIGNATURE_MIN_KEYSGROUP,
+} from './constants';
 import {
 	prependPlusToPublicKeys,
 	validateKeysgroup,
 	wrapTransactionCreator,
+	isValidInteger,
 } from './utils';
 
-const registerMultisignatureAccount = ({ keysgroup, lifetime, minimum }) => {
+const validateInputs = ({ keysgroup, lifetime, minimum }) => {
+	if (
+		!isValidInteger(lifetime) ||
+		lifetime < MULTISIGNATURE_MIN_LIFETIME ||
+		lifetime > MULTISIGNATURE_MAX_LIFETIME
+	) {
+		throw new Error(
+			`Please provide a valid lifetime value. Expected integer between ${MULTISIGNATURE_MIN_LIFETIME} and ${MULTISIGNATURE_MAX_LIFETIME}.`,
+		);
+	}
+
+	if (
+		!isValidInteger(minimum) ||
+		minimum < MULTISIGNATURE_MIN_KEYSGROUP ||
+		minimum > MULTISIGNATURE_MAX_KEYSGROUP
+	) {
+		throw new Error(
+			`Please provide a valid minimum value. Expected integer between ${MULTISIGNATURE_MIN_KEYSGROUP} and ${MULTISIGNATURE_MAX_KEYSGROUP}.`,
+		);
+	}
+
+	if (keysgroup.length < minimum) {
+		throw new Error(
+			`Expected minimum ${minimum} public keys in the keysgroup.`,
```

Comment:
This message makes an assumption about what went wrong. Something like this would be better: `Minimum number of signatures is larger than the number of keys in the keysgroup.`